packages/client/src/layers/noa/systems/createSoundSystem.ts

```diff
@@ -34,9 +34,9 @@ export function createSoundSystem(network: NetworkLayer, context: NoaLayer) {
   const scene: Scene = noa.rendering.getScene();
   const musicUrls = [
     "/audio/OP_World_Theme_Mix_1.mp3",
-    "/audio/Berceuse._Mix_1.mp3",
-    "/audio/Gymnopedia_Mix_2.mp3",
-    "audio/Mazurka Op.25 No. 2_Mix_1.mp3",
+    "/audio/Berceuse._Mix_2.mp3",
```

Comment:
```suggestion
    "/audio/Berceuse_Mix_2.mp3",
```

seems like the dot is gone from the file name